### PR TITLE
feat: home page added.

### PR DESCRIPTION
**Pull request by krishnakant01**
I've created the home page. There are 26 files out of which 24 are image files.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -1,6 +1,101 @@
...
+}
+
+.search-input {
+    border: none;
+    width: calc(100%-2rem);
+    background-color: transparent;
+}
+
+.search-input:focus {
+    outline: none;
```
Comment (dev-enforced) [inline style.css:46]: Alloting outline:none is not a preferred practice instead you can use a value of 1px solid transparent

**Review by dev-enforced — APPROVED** [review]
image
Do change the color of text when viewed on mobile . It looks too much bright unable to read clearly

File: style.css [hunk 2]
```diff
@@ -1,6 +1,101 @@
...
+@import url('https://fonts.googleapis.com/css2?family=Open+Sans:wght@800&display=swap');
+
+* {
+    box-sizing: border-box;
+}
+
+body {
+    margin: 0;
+    padding: 0;
+}
```
Comment (rkapoor10) [inline style.css:10]: Why to give Reset CSS and these properties if you are using component lib which may already have them ?

**Review by rkapoor10 — APPROVED** [review]
Code is well structured looks good to merge. Try to use as many classes from component lib as possible.

File: style.css (hunk 2 above)
Comment (krishnakant01) [reply style.css:10]: Yes, I was just ensuring ki its reset to avoid unexpected outputs

**Comment by krishnakant01** [comment]
> 

It's not mobile responsive yet, I'll keep the suggestion in mind

File: style.css (hunk 1 above)
Comment (krishnakant01) [reply style.css:46]: Okay thanks for the suggestion!

**Pull request merged**